src/api/VoteMonitor.Api.Observer/Controllers/ObserverController.cs

```diff
@@ -42,17 +42,59 @@ public async Task<List<ObserverModel>> GetObservers(ObserverListQuery query)
         public void Import(IFormFile file, [FromForm] object a)
         { }
 
+        /// <summary>
+        ///  Adds an observer.
+        /// </summary>
+        /// <param name="model"></param>
+        /// <returns>Boolean indicating whether or not the observer was added successfully.</returns>
         [Authorize]
         [HttpPost]
         [Route("")]
+        [Produces(type: typeof(bool))]
         public async Task<dynamic> NewObserver(NewObserverModel model)
```

Comment:
could this have been modified as following (and then the `ProducesAttribute` wouldn't have been required)? 
```suggestion
        public async Task<bool> NewObserver(NewObserverModel model)
```